src/test/java/com/cloudant/tests/ReplicateBaseTest.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
If feature-http-replacement has already been rebased on master then I guess you can skip this commit (or maybe skip it anyway as it didn't seem to help your issue).